docs-site/src/css/custom.css

```
@@ -56,6 +56,10 @@
   --ifm-leading: 1.75;
 }
 
+/* Responsive overrides must be imported after the base :root block so that
+   media-query scoped --ifm-spacing-horizontal values take precedence. */
+@import "./responsive.css";
```

Biome (2.4.4)

[error] This `@import` is in the wrong position.

(lint/correctness/noInvalidPositionAtImportRule)